### DISCUSSION TITLE
Implement option to disable 'deep link' feature

### DISCUSSION THREAD

**Discussion by itsjusth** [grafana/grafana#73292] [Data Sources]
In the Azure Monitor plugin, all panels become automatically hyperlinked with the ability for the user to click and "View in Azure Portal".  This feature is almost _never_ what we want the user to do.  In many cases, one or two columns will be hyperlinked to go to a specific record in another system such as Azure DevOps or Service Now.  The users cannot tell that these columns are specially hyperlinked because _every_ column is hyperlinked.  The best implementation for us would be to have this option disabled by default and only enabled in the panel editor if the editor wants it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82567. Feel free to continue the discussion around this there. Thank you!
